src/System.Private.Xml/tests/XmlSchema/XmlSchemaSet/TC_SchemaSet_AnyAttribute.cs

```diff
@@ -0,0 +1,202 @@
+﻿using Xunit;
+using Xunit.Abstractions;
+using System.IO;
+using System.Xml.Schema;
+
+namespace System.Xml.Tests
+{
+    public class TC_SchemaSet_AnyAttribute : TC_SchemaSetBase
+    {
+        private ITestOutputHelper _output;
+        public bool bWarningCallback;
+        public bool bErrorCallback;
+        public int errorCount;
+        public int warningCount;
+        public bool WarningInnerExceptionSet = false;
+        public bool ErrorInnerExceptionSet = false;
```

Comment:
The same question as above. Should i use project code style or correct code style? It will be ok if this class will be implemented in different code style than others?